package-lock.json

```diff
@@ -458,7 +524,7 @@
     "absolute": {
       "version": "0.0.1",
       "resolved": "https://registry.npmjs.org/absolute/-/absolute-0.0.1.tgz",
-      "integrity": "sha512-Zs3Sz3lmEIH76fvnc4jto4CpuI5bhU2TD+jtBYVIfXnjFm+49y1opKxNy4gtNF+xSLBL7hXRpLwypc+o2MzBIQ==",
+      "integrity": "sha1-wigi+H4ck59XmIdQTZwQnEFzgp0=",
```

Comment:
Looks like some SHA-512 hashes somehow got downgraded to SHA-1 :S.

I don't think that'll be show-stopper, but wanted to point it out just in case.

Reply:
that is fine